launch.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+printf "\n-- stoping docker hasura_graphql-engine_1 hasura_postgres_1 auth-server_postgres_1\n"
+docker stop hasura_graphql-engine_1 hasura_postgres_1 auth-server_postgres_1;
+
+printf "\n\n-- deleting docker hasura_graphql-engine_1 hasura_postgres_1 auth-server_postgres_1\n"
+docker rm hasura_graphql-engine_1 hasura_postgres_1 auth-server_postgres_1;
+
+printf "\n\n-- killing any node running\n"
+pkill -f node;
+
+printf "\n\n-- launching auth-server docker\n"
+cd auth-server && docker-compose up -d && cd ..;
+
+sleep 5
+
+printf "\n\n-- launching auth-server\n" 
+cd auth-server && yarn start &> auth.log &
+
+sleep 5
+
+printf "\n\n-- migrating auth-server\n" 
+cd auth-server && knex migrate:latest && cd ..;
```

Comment:
knex migrate:latest can be added to package.json start script as well. It will help with k8s as well

Reply:
right, it was there already actually `yarn migrate`